fastkml/times.py

```diff
@@ -206,6 +210,7 @@ def __repr__(self) -> str:
             f"id={self.id!r}, "
             f"target_id={self.target_id!r}, "
             f"timestamp={self.timestamp!r}, "
+            f"when={self.when!r}, "
```

Comment:
There is no need to add both `when` and `timestamp`, as the `when` object can have more attributes like `id` and `targetId` it will be better to return just this.

Reply:
I think, give that we are now using `point_in_time` which is of type `KmlDateTime | When`, we can simply print its value.